packages/bierzo-wallet/src/routes/login/index.tsx

```diff
@@ -34,6 +36,10 @@ const Login = (): JSX.Element => {
     const chainTokens = await getTokens();
     dispatch(addTickersAction(chainTokens));
 
```

Comment:
Here I get the keys from the store and I use them for getting positive account balances.